Configuration/HLT/python/addOnTestsHLT.py

```diff
@@ -22,13 +22,13 @@ def addOnTestsHLT():
                                'HLTrigger/Configuration/test/OnLine_HLT_PRef.py',
                                'cmsDriver.py RelVal -s HLT:PRef,RAW2DIGI,L1Reco,RECO --mc --scenario=pp -n 10 --conditions auto:run3_mc_PRef --relval 9000,50 --datatier "RAW-HLT-RECO" --eventcontent FEVTDEBUGHLT --customise=HLTrigger/Configuration/CustomConfigs.L1THLT --customise=HLTrigger/Configuration/CustomConfigs.HLTRECO --era Run3 --processName=HLTRECO --filein file:RelVal_Raw_PRef_MC.root --fileout file:RelVal_Raw_PRef_MC_HLT_RECO.root'],
-                    'hlt_data_Fake' : ['cmsDriver.py RelVal -s L1REPACK:GT1   --data --scenario=pp -n 10 --conditions auto:run1_hlt_Fake --relval 9000,50 --datatier "RAW" --eventcontent RAW --customise=HLTrigger/Configuration/CustomConfigs.L1T --fileout file:RelVal_Raw_Fake_DATA.root --filein /store/data/Run2012A/MuEG/RAW/v1/000/191/718/14932935-E289-E111-830C-5404A6388697.root',
+                    'hlt_data_Fake' : ['cmsDriver.py RelVal -s L1REPACK:GT1 --data --scenario=pp -n 10 --conditions auto:run1_hlt_Fake --relval 9000,50 --datatier "RAW" --eventcontent RAW --customise=HLTrigger/Configuration/CustomConfigs.L1T --fileout file:RelVal_Raw_Fake_DATA.root --filein root://eoscms.cern.ch//eos/cms/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2012A_MuEG_run191718/14932935-E289-E111-830C-5404A6388697.root',
                                'HLTrigger/Configuration/test/OnLine_HLT_Fake.py',
                                'cmsDriver.py RelVal -s HLT:Fake,RAW2DIGI,L1Reco,RECO --data --scenario=pp -n 10 --conditions auto:run1_data_Fake --relval 9000,50 --datatier "RAW-HLT-RECO" --eventcontent FEVTDEBUGHLT --customise=HLTrigger/Configuration/CustomConfigs.L1THLT --customise=HLTrigger/Configuration/CustomConfigs.HLTRECO --processName=HLTRECO --filein file:RelVal_Raw_Fake_DATA.root --fileout file:RelVal_Raw_Fake_DATA_HLT_RECO.root'],
-                    'hlt_data_Fake1': ['cmsDriver.py RelVal -s L1REPACK:GCTGT --data --scenario=pp -n 10 --conditions auto:run2_hlt_Fake1 --relval 9000,50 --datatier "RAW" --eventcontent RAW --customise=HLTrigger/Configuration/CustomConfigs.L1T --era Run2_25ns --fileout file:RelVal_Raw_Fake1_DATA.root --filein /store/data/Run2015D/MuonEG/RAW/v1/000/256/677/00000/80950A90-745D-E511-92FD-02163E011C5D.root',
+                    'hlt_data_Fake1': ['cmsDriver.py RelVal -s L1REPACK:GCTGT --data --scenario=pp -n 10 --conditions auto:run2_hlt_Fake1 --relval 9000,50 --datatier "RAW" --eventcontent RAW --customise=HLTrigger/Configuration/CustomConfigs.L1T --era Run2_25ns --fileout file:RelVal_Raw_Fake1_DATA.root --filein root://eoscms.cern.ch//eos/cms/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root',
```

Comment:
Why do you pre-pend ```root://eoscms.cern.ch//eos/cms``` ?

Reply:
I don't think it's necessary, so we could change it.

I just tried to be consistent with `cmsDriver.csh` (where we already some vars using the `root://eoscms.cern.ch//eos/cms/` prefix), and I found this useful to specify where the single copy of this file actually is (it is only in that one place). Also, I see that using the explicit prepend avoids failed attempts like [*] when not testing in the CERN network.


[*]
Outside CERN without EOS specifier:
```
Failed open file in the dCache.
%MSG-w StorageFactory::stagein():  PoolSource:source@sourceConstruction  19-Oct-2022 14:18:31 CEST pre-events
Failed to stage in file 'dcap://t3se01.psi.ch/pnfs/psi.ch/cms/trivcat/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root' because:
An exception of category 'FileStageInError' occurred while
   [0] Calling DCacheStorageMaker::stagein()
Exception Message:
Cannot stage in file 'dcap://t3se01.psi.ch/pnfs/psi.ch/cms/trivcat/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root', error was: "No such file or directory" (dc_errno=30)

%MSG
19-Oct-2022 14:18:31 CEST  Initiating request to open file dcap://t3se01.psi.ch/pnfs/psi.ch/cms/trivcat/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root
Failed open file in the dCache.
19-Oct-2022 14:18:32 CEST  Initiating request to open file root://t3se01.psi.ch:1094//store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root
%MSG-w XrdAdaptorInternal:  file_open 19-Oct-2022 14:18:33 CEST pre-events
Failed to open file at URL root://t3se01.psi.ch:1094//store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root?xrdcl.requuid=b93f8773-a4a0-4986-9567-2aa83ec62651.
%MSG
%MSG-w XrdAdaptorInternal:  file_open 19-Oct-2022 14:18:33 CEST pre-events
Failed to open file at URL root://t3se01.psi.ch:1094//store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root?tried=&xrdcl.requuid=8e44bdf4-07eb-4577-bbb0-651bab6809c7.
%MSG
19-Oct-2022 14:18:33 CEST  Initiating request to open file root://xrootd-cms.infn.it//store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root
19-Oct-2022 14:18:36 CEST  Successfully opened file root://xrootd-cms.infn.it//store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root
```
With EOS specifier:
```
19-Oct-2022 14:14:42 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root
19-Oct-2022 14:14:48 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/RAW/Run2015D_MuonEG_run256677/80950A90-745D-E511-92FD-02163E011C5D.root
```

Reply:
OK, this is very useful!